### ISSUE TITLE
Add Wawsome overlay to the list of overlays

### ISSUE THREAD

**Issue by teodorachiosa** [karlgroves/overlayfactsheet#1369]
Hey, I'd like to add a new overlay, Wawsome, that bought multiple domains, including [wawsome.com](http://wawsome.com) and [wcag.ro](http://wcag.ro).
I'll create the PR myself.

Please close this issue if it's not allowed.

Red flags at a quick glance:
- The homepage doesn't feature visible focus states for the top navigation.
- On the [Pricing](https://www.wawsome.com/pricing-page) page, the overlay button is at the end of the DOM structure and doesn't have a descriptive label. The tabs also don't have a visible focus indicator.
- On the [FAQ](https://www.wawsome.com/resources/faq) page, the accordions aren't keyboard accessible.
